src/lib.rs

```diff
@@ -179,7 +179,10 @@ impl Surreal {
     pub async fn query(&self, sql: String, bindings: Option<Value>) -> Result<Value> {
         let mut response = match bindings {
             None => self.db.query(sql).await.map_err(err_map)?,
-            Some(b) => self.db.query(sql).bind(b).await.map_err(err_map)?,
+            Some(b) => {
+                let b = json(&b.to_string()).unwrap();
```

Comment:
I know it's unlikely to run into an error here but it's possible. Kindly propagate it please.